src/main/java/com/rcv/CastVoteRecord.java

```diff
@@ -70,6 +70,30 @@
     sortRankings(rankings);
   }
 
+  // function: equals
+  // purpose: validate that two CVRs are equivalent for test purposes
+  // param: left CastVoteRecord to test for equality
+  // param: right CastVoteRecord to test for equality
+  // returns: true if left CastVoteRecord is logically equal to right CastVoteRecord
+  static boolean equalsCVR(CastVoteRecord left, CastVoteRecord right) {
```

Comment:
Just noticed that this is unused. Maybe save it for a future PR?

Reply:
Right.  I discarded the test code that used this, but I expect to bring it back once we have JUnit running.  I guess I could remove it.  You think it matters?